docs/develop/java/temporal-nexus.mdx

```diff
@@ -0,0 +1,654 @@
+@ServiceImpl(service = NexusService.class)
+public class NexusServiceImpl {
+  @OperationImpl
+  public OperationHandler<NexusService.EchoInput, NexusService.EchoOutput> echo() {
+    // WorkflowClientOperationHandlers.sync is a meant for exposing simple RPC handlers.
+    return WorkflowClientOperationHandlers.sync(
+        // The method is provided with an SDK client that can be used for arbitrary calls such as
+        // signaling, querying,
+        // and listing workflows but implementations are free to make arbitrary calls to other
+        // services or databases, or
+        // perform simple computations such as this one.
+        (ctx, details, client, input) -> new NexusService.EchoOutput(input.getMessage()));
+  }
+// ...
+}
+```
+
+<!--SNIPEND-->
+
+### Develop an Asynchronous Nexus Operation handler to start a Workflow
+
+Use the `WorkflowClientOperationHandlers.fromWorkflowMethod` method, which is the easiest way to expose a Workflow as an operation.
+See alternatives [here](https://pkg.go.dev/go.temporal.io/sdk/temporalnexus).
+
+<!--SNIPSTART samples-java-nexus-handler {"selectedLines": ["1-5", "18-37"]}-->
+
+[core/src/main/java/io/temporal/samples/nexus/handler/NexusServiceImpl.java](https://github.com/temporalio/samples-java/blob/nexus-snip-sync/core/src/main/java/io/temporal/samples/nexus/handler/NexusServiceImpl.java)
+
+```java
+// To create a service implementation, annotate the class with @ServiceImpl and provide the
+// interface that the service implements. The service implementation class should have methods that
+// return OperationHandler that correspond to the operations defined in the service interface.
+@ServiceImpl(service = NexusService.class)
+public class NexusServiceImpl {
+// ...
+  @OperationImpl
+  public OperationHandler<NexusService.HelloInput, NexusService.HelloOutput> hello() {
+    // Use the WorkflowClientOperationHandlers.fromWorkflowMethod constructor, which is the easiest
+    // way to expose a workflow as an operation.
+    return WorkflowClientOperationHandlers.fromWorkflowMethod(
+        (ctx, details, client, input) ->
+            client.newWorkflowStub(
+                    HelloHandlerWorkflow.class,
+                    // Workflow IDs should typically be business meaningful IDs and are used to
+                    // dedupe workflow starts.
+                    // For this example, we're using the request ID allocated by Temporal when the
+                    // caller workflow schedules
+                    // the operation, this ID is guaranteed to be stable across retries of this
+                    // operation.
+                    //
+                    // Task queue defaults to the task queue this operation is handled on.
+                    WorkflowOptions.newBuilder().setWorkflowId(details.getRequestId()).build())
+                ::hello);
+  }
+}
+```
+
+<!--SNIPEND-->
+
+Workflow IDs should typically be business meaningful IDs and are used to dedupe Workflow starts.q
```

Comment:
There is another line here in the Go SDK docs

https://docs.temporal.io/develop/go/nexus#develop-an-asynchronous-nexus-operation-handler-to-start-a-workflow

> For the HelloOperation, input.ID is passed as part of the Nexus Service contract.

And I could not figure out what it was trying to say

Reply:
gotcha added a note on this